src/components/UI/organisms/AddMealForm/AddMealForm.jsx

```diff
@@ -0,0 +1,77 @@
+import { Box, Button, FormHelperText, Stack, TextField, Typography } from '@mui/material';
+import { addMealSchema } from '../../../../schemas/addMealSchema';
+import { useForm } from 'react-hook-form';
+import './AddMealForm.css';
+import { yupResolver } from '@hookform/resolvers/yup';
+
+function AddMealForm() {
+  const {
+    register,
+    handleSubmit,
+    formState: { errors },
+  } = useForm({
+    mode: 'onBlur',
+    resolver: yupResolver(addMealSchema),
+  });
+
+  const onSubmit = (data) => console.log(data);
+  return (
+    <>
+      <Typography className="AddMealForm__header">Add Meal Form</Typography>
+      <Box className="AddMealForm" component="form" onSubmit={handleSubmit(onSubmit)}>
+        <Stack spacing={2}>
+          <TextField
+            error={errors?.title}
+            label="Meal Title"
+            placeholder="Meal title"
+            required
+            type="text"
+            autoComplete="Meal-title"
+            {...register('title')}
+          />
+          {errors?.title ? <FormHelperText error>{errors?.title && errors?.title.message}</FormHelperText> : null}
+          <TextField
+            error={!!errors?.description}
```

Comment:
Why '!!errors?.NAME'?

Reply:
Mui needs a boolean to show an error. https://mui.com/material-ui/api/text-field/
![obraz](https://user-images.githubusercontent.com/80857044/169405645-c6dbe650-a175-47dd-b457-ea786619c136.png)

Reply:
You can use Boolean constructor. There are more verbose and when seeing 'Boolean' in code everyone just know from the start what it stands for. Without even looking at the content. 

'Boolean(errors?.NAME)'